packages/website/components/account/filesManager/filesManager.js

```diff
@@ -116,8 +116,8 @@ const FilesManager = ({ className, content, onFileUpload }) => {
     }
   }, [fetchPinsDate, listPinned, isFetchingPinned, apiToken]);
   useEffect(() => {
-    getTokens()
-  }, []);
+    getTokens();
+  }, [getTokens]);
```

Comment:
This was the only linting violation I had to manually resolve, the rest were all minor things it seems. I don't think this breaks anything here?

Reply:
No, not seeing any issues here either